src/main.ts

```diff
@@ -25,8 +26,8 @@ export default class FolderIndexPlugin extends Plugin {
 		this.settings.hideIndexFiles = false;
 		await this.saveSettings();
 
-		this.oldGraphSetting = this.settings.graphOverwrite
-
+		this.oldGraphSetting = this.settings.graphOverwrite;
+		this.includeFileContent = this.settings.includeFileContent;
```

Comment:
This is not required within the main class. I just had todo some sneaky stuff because of me overwriting the Graph

Reply:
Removed, thanks